### ISSUE TITLE
Set primary monitors in virtual profiles

### ISSUE THREAD

**Issue by Vladimir-csp** [phillipberndt/autorandr#103]
When applying virtual profiles, set primary monitor. Prioritize the largest resolution, then the largest physical dimensions.

**Comment by phillipberndt** [comment]
Sure, why not, sounds reasonable. Do you want to implement this or shall I consider this a feature request?

**Comment by Vladimir-csp** [comment]
It's a feature request, I do not comprehend much of the code yet.

BTW, is there a specific reason for relative xrandr options like `--right-of`, `--same-as`  not being used for virtual profiles? Could simplify things.

**Comment by phillipberndt** [comment]
> BTW, is there a specific reason for relative xrandr options like --right-of, --same-as not being used for virtual profiles? Could simplify things.

Yes: autorandr compares old/new profiles before/after loading to check whether (a) it is necessary to run xrandr at all (b) to check whether the changes were applied successfully.
